.ci/check-packages.groovy

```diff
@@ -31,11 +27,10 @@ pipeline {
     rateLimitBuilds(throttle: [count: 60, durationName: 'hour', userBoost: true])
   }
   triggers {
-    cron '@weekly'
+    upstream("apm-server/apm-server-mbp/${env.JOB_BASE_NAME}")
```

Comment:
Do we want to trigger it as upstream of PRs?

Reply:
I configured the JJBB to only listen for branches:
- https://github.com/elastic/apm-server/pull/4401/files#r520552689
- https://github.com/elastic/apm-server/pull/4401/files#diff-6b33f933d29b0a0317ef241367a3bfc925208a9b0cfce83c1872f6e99c1e2e97R29-R30

It should not be a problem, right?

Reply:
yep, in that case, is fine